code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -409,6 +409,7 @@
 
 	var/msg = "<span class='info'>*---------*\nThis is [icon2html(src, user)] \a <EM>[src]</EM>!\n"
 	if (src.stat == DEAD)
+		msg += surgeries_examine()
```

Comment:
This should probably show for live animals too.

Reply:
Only ded slimes can get surgery but okay